### DISCUSSION TITLE
I2C, Serial, SPI Protcols

### DISCUSSION THREAD

**Discussion by chemistorge** [fhdm-dev/scoppy#97] [General]
If you could make the app analyse and display I2C, Serial and Spi communication protocols, I would gladly pay more for the app.

**Comment by fhdm-dev** [comment]
Hi,
That is something I would like to add in the long term. In the near future the best I can do is add an option to the export function to make the exported data more compatible with sigrok/pulseview. Hopefully this won't be too far away.

**Comment by chemistorge** [comment]
Ok, thank you, I look forward to seeing updates.

Regards
